u-mode-api/src/cert.rs

```diff
@@ -0,0 +1,28 @@
+use attestation::MSMT_REGISTERS;
+use data_model::DataInit;
+
+/// CDI ID length.
+pub const CDI_LEN: usize = 20;
```

Comment:
`attestation/src/manager.rs` says this is 32

Reply:
No, that's the CDI length. The CDI ID (generated by the public key) is 20. Defined in rice/cdi.rs CDI_ID_LEN.

Reply:
Changed to the less confusing name CDI_ID_LEN.